projects/uswds-components/src/lib/table/table.component.html

```diff
@@ -1,5 +1,5 @@
 <div [ngClass]="{'usa-table-container--scrollable': scrollable}">
-  <table role="table" class="usa-table" 
+  <table role="table" class="usa-table full-width" 
```

Comment:
The table should not be full width at all times.

Reply:
So looks like this does not force that style onto the table, now. However, there still isn't a need to have this class applied at all times.